### PR TITLE
Add support for outputting JUnit XML

### LINKED ISSUE #8: Output JUnit XML files
For better integration with CI

### PR DESCRIPTION
**Pull request by rehandalal**
Fixes #8 

r?

### CODE REVIEW

File: therapist/runner/sets.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class Set(object):
```
Comment (mythmon) [inline therapist/runner/sets.py]: This name is sort of confusing, given the Python builtin `set`.

Comment (rehandalal) [inline therapist/runner/sets.py]: /me nods. maybe `Group`?

Comment (mythmon) [inline therapist/runner/sets.py]: They both sound better. `Collection` feels more apt, but `Group` is shorter.

**Comment by mythmon** [comment]
I skimmed the code, and I think it all looks fine. I ran it locally, and it seems to work. r+

**Pull request merged**